src/queryAnalysis/utils.go

```diff
@@ -76,6 +83,14 @@ func BindQueryResults(rows *sqlx.Rows, queryDetailsDto models.QueryDetailsDto) (
 			modelIngestor.StatementType = model.StatementType
 			modelIngestor.CollectionTimestamp = model.CollectionTimestamp
 			results = append(results, modelIngestor)
+
+			// fetch and generate execution plan
+			wg.Add(1)
```

Comment:
can we remove goroutines for now, because ingestions is first priority to complete with aysnc ingestion will create problem